app/model/model.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/mizkei/self-lint-sample/app/data"
+	"github.com/mizkei/self-lint-sample/app/test"
```

Comment:
**[self-lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
forbidden to import github.com/mizkei/self-lint-sample/app/test